qemu/tests/cfg/differential_backup.cfg

```diff
@@ -0,0 +1,27 @@
+- differential_backup:
+    virt_test_type = qemu
+    type = differential_backup
+    only Linux
```

Comment:
Why set "only linux" here? In fact, differential_backup won't limited to linux, windows is also supported.